packages/pygobject.rb

```diff
@@ -3,31 +3,34 @@
 class Pygobject < Package
   description 'PyGObject is a Python package which provides bindings for GObject based libraries such as GTK+, GStreamer, WebKitGTK+, GLib, GIO and many more.'
   homepage 'https://pygobject.readthedocs.io/'
-  @_ver = '3.42.0'
-  version "#{@_ver}-1"
+  @_ver = '3.42.2'
+  version @_ver.to_s
```

Comment:
Can we get rid of this ridiculous rubocop `.to_s` madness?

Reply:
Changed in all packages.